policyengine_us/parameters/gov/states/mo/tax/income/minimum_taxable_income.yaml

```diff
@@ -6,7 +6,7 @@ values:
   2020-01-01: 106
   2021-01-01: 108
   2022-01-01: 111
-  2023-01-01: 1_000
+  2023-01-01: 1_207
```

Comment:
Is this because the legal code is 1000 but indexed from some date?

Reply:
This is how we are reading the legal code:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/798949b2-045c-407e-96d5-9b713c942685)